pkg/cli/registry/login/login.go

```diff
@@ -1,18 +1,16 @@
 package login
 
 import (
-	"bytes"
 	"context"
-	"encoding/json"
 	"fmt"
-	"io"
-	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/containers/image/v5/pkg/docker/config"
+	imageTypes "github.com/containers/image/v5/types"
```

Comment:
Nit: most import aliases are lowercase to differentiate that from variables.

Reply:
Can we maybe make these imports be:
dockerconfig and containerstypes so it's explicit where these are coming while reading the code